raiden/network/proxies/payment_channel.py

```diff
@@ -26,11 +29,35 @@ def detail(self) -> Dict:
         """ Returns the channel details. """
         return self.token_network.detail(self.partner_address)
 
-    def settle_block_number(self) -> typing.BlockNumber:
-        """ Returns the settle block of the channel"""
+    def settle_block_number(self) -> int:
+        """ Returns the channels settle block number.
+
+        This is relative while the channel is open and becomes absolute when the channel is closed
```

Comment:
> This is relative while the channel is open and becomes absolute when the channel is closed

this inconsistency can lead to many errors. Does other code depend on the relative number? If absolute block no. can be computed for both open & closed channel, then I'd make the method return it every time.

Reply:
you probably can - it's one-liner that can be added later anytime.